.gitignore

```diff
@@ -3,7 +3,12 @@ __pycache__
 .history
 .hypothesis/
 .tox/
-build/
+build/*
+!build/deployments/
+build/deployments/*
```

Comment:
This is duplicated no? 

Is it that you want to exclude `build/deployments` and then make specific exceptions for `build/deployments/map.json` , `build/deployments/1`, `build/deployments/137`. If so, then L7 (`!build/deployments/`) can probably go away.

Reply:
^ Ping @cygnusv . More broadly, PR needs a rebase.